src/astroport/utils.rs

```diff
@@ -809,88 +854,4 @@ mod tests {
         println!("LP token balance: {:?}", lp_token_balance);
         assert!(lp_token_balance.balance > Uint128::zero());
     }
-
-    #[test]
-    fn test_create_astroport_pair() {
-        let app = OsmosisTestApp::new();
-        let admin = app
-            .init_account(&[Coin::new(100000000000000000u128, "uosmo")])
-            .unwrap();
-
-        let contract_types = get_local_artifacts()
-            .into_iter()
-            .map(|(name, artifact)| (name, ContractType::Artifact(artifact)))
-            .collect();
-
-        // Instantiate contracts
-        let contracts = setup_astroport(&app, contract_types, &admin);
-
-        // Create XYK pool
-
-        let asset_infos: [AssetInfo; 2] = [
-            AssetInfo::NativeToken {
-                denom: "uosmo".into(),
-            },
-            AssetInfo::NativeToken {
-                denom: "uatom".into(),
-            },
-        ];
-
-        create_astroport_pair(
-            &app,
-            &contracts.factory.address,
-            PairType::Xyk {},
-            asset_infos,
-            None,
-            &admin,
-            None,
-        );
-    }
-
-    #[test]
-    fn test_create_astroport_pair_with_initial_liquidity() {
-        let app = OsmosisTestApp::new();
-        let admin = app
-            .init_account(&[
-                Coin::new(100000000000000000u128, "uosmo"),
-                Coin::new(100000000000000000u128, "uatom"),
-            ])
-            .unwrap();
-
-        let contract_types = get_local_artifacts()
-            .into_iter()
-            .map(|(name, artifact)| (name, ContractType::Artifact(artifact)))
-            .collect();
-
-        // Instantiate contracts
-        let contracts = setup_astroport(&app, contract_types, &admin);
-
-        // Create XYK pool
-        let asset_infos: [AssetInfo; 2] = [
-            AssetInfo::NativeToken {
-                denom: "uosmo".into(),
-            },
-            AssetInfo::NativeToken {
-                denom: "uatom".into(),
-            },
-        ];
-
-        let (pool, _lp) = create_astroport_pair(
-            &app,
-            &contracts.factory.address,
-            PairType::Xyk {},
-            asset_infos,
-            None,
-            &admin,
-            Some([1000000u128.into(), 1000000u128.into()]),
-        );
-
-        // Query pool info
-        let wasm = Wasm::new(&app);
-        let pool_info: PoolResponse = wasm
-            .query(&pool, &astroport::pair::QueryMsg::Pool {})
-            .unwrap();
-        assert_eq!(pool_info.assets[0].amount, Uint128::from(1000000u128));
-        assert_eq!(pool_info.assets[1].amount, Uint128::from(1000000u128));
-    }
```

Comment:
why remove these tests?

Reply:
Was too lazy to fix them, and they also exist in the AstroportTestRobot tests